crimsobot/cogs/reactions.py

```diff
@@ -0,0 +1,183 @@
+from tortoise.exceptions import DoesNotExist
+
+from config import ADMIN_USER_IDS
+from crimsobot.bot import CrimsoBOT
+from crimsobot.models.fun_facts import FunFacts, NoFactsExist
+from crimsobot.utils import tools as c
+
+log = logging.getLogger(__name__)
+
+
+class CleanMentions(commands.Converter):
+    async def convert(self, ctx: commands.Context, string: str) -> str:
+        """Clean up those silly mention inconsistencies across platforms."""
+        for mention in ctx.message.mentions:
+            string = string.replace(f'<@!{mention.id}>', f'<@{mention.id}>', 1)
+
+        return string
+
+
+class Reactions(commands.Cog):
+    def __init__(self, bot: CrimsoBOT):
+        self.bot = bot
+
+    @commands.group(aliases=['facts'])
+    async def fact(self, ctx: commands.Context) -> None:
+        """Testing out facts."""
+        return
+
+    @fact.command(name='about')
+    async def fact_about(self, ctx: commands.Context, *, subject: CleanMentions) -> None:
+        """Write up some HELP"""
+        try:
+            fact_object = await FunFacts.get_by_subject(subject.lower().strip(), ctx.guild.id)
+        except NoFactsExist:
+            embed = c.crimbed(
+                title='OMAN',
+                descr=f'There are no facts about {subject}!',
+                footer='You can add one by using ">fact about [fact subject]; [some fun fact]"',
+                color_name='yellow',
+                thumb_name='weary',
+            )
+            await ctx.send(embed=embed, delete_after=18)
+            return
+
+        return_string = ''.join([
+            f'`Fact ID: {fact_object.uuid}` · ',
+            f"**Here's a fact about {fact_object.subject.upper()}**:\n\n",
+            f'{fact_object.funfact}\n\n',
+        ])
+
+        await ctx.send(return_string)
+
+    @fact.command(name='add')
+    async def fact_add(self, ctx: commands.Context, *, something: CleanMentions) -> None:
+        """Add a fact"""
+
+        # if there are any errors in user input, use this embed
+        error_embed = c.crimbed(
+            title='BAD AT COMPUTER',
+            descr="You didn't do that correctly! Try this:\n`>fact add your subject; your fact`",
+            footer='You can always check ">help fact" for more info!',
+            color_name='yellow',
+            thumb_name='weary',
+        )
+
+        # see if there are uploaded files or media
+        try:
+            link_list = []
+            for link in ctx.message.attachments:
+                link_list.append(link.url)
+            links = '\n'.join(link_list)
+        except IndexError:
+            links = ''
+
+        user_input = something.split(';', 1)
+
+        fact_subject = user_input.pop(0).strip().lower()
+        try:
+            fact = f'{user_input[0].strip()} {links}'
+        except IndexError:
+            await ctx.send(embed=error_embed, delete_after=18)
+            return
+
+        if len(fact_subject) < 1 or len(fact) < 1:
```

Comment:
Empty strings are falsey values, so something like 
```suggestion
        if not (fact_subject or fact):
```
would work here.